controller/users.go

```diff
@@ -215,9 +215,43 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 func (c *UsersController) List(ctx *app.ListUsersContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
 		var err error
-		var users []*account.User
 		var result *app.UserArray
-		users, err = appl.Users().List(ctx.Context)
+
+		if ctx.Username != nil {
+			// From a data model perspective, we are querying by identity ( and not user )
+			identities, err := appl.Identities().Query(account.IdentityFilterByUsername(*ctx.Username))
```

Comment:
Got it!

Btw, thanks for pointing this out - this is a saviour ` account.IdentityWithUser()`